### PR TITLE
store/tikv: extract methods for LockCtx

### PR DESCRIPTION
**Pull request by disksing**
Signed-off-by: disksing <i@disksing.com>

### What problem does this PR solve?
To reduce direct dependency on store/tikv/kv

Part of https://github.com/pingcap/tidb/issues/22513

### What is changed and how it works?
What's Changed:
- extract some methods for LockCtx so don't have to use fields directly.

### Check List 
Tests 
- Unit test

### Release note 
- No release note

### CODE REVIEW

File: executor/point_get.go [hunk 1]
```diff
@@ -322,19 +321,14 @@ func (e *PointGetExecutor) lockKeyIfNeeded(ctx context.Context, key []byte) erro
 	if e.lock {
 		seVars := e.ctx.GetSessionVars()
 		lockCtx := newLockCtx(seVars, e.lockWaitTime)
-		lockCtx.ReturnValues = true
-		lockCtx.Values = map[string]tikvstore.ReturnedValue{}
+		lockCtx.InitReturnValues(1)
```
Comment (AndreMouche) [inline executor/point_get.go:324]: could we use 0 instead?

**Review by AndreMouche — APPROVED** [review]
LGTM

File: executor/point_get.go (hunk 1 above)
Comment (disksing) [reply executor/point_get.go:324]: Actually, I'm not very sure either. /cc @sticnarf could help to take a look?

Comment (sticnarf) [reply executor/point_get.go:323]: `1` looks reasonable to me.

**Review by sticnarf — APPROVED** [review]
LGTM

**Pull request merged**